staging/src/k8s.io/apiserver/pkg/server/options/audit.go

```diff
@@ -111,6 +111,13 @@ type AuditWebhookOptions struct {
 func NewAuditOptions() *AuditOptions {
 	defaultLogBatchConfig := pluginbuffered.NewDefaultBatchConfig()
 	defaultLogBatchConfig.ThrottleEnable = false
+	// Set the max batch size to 1, and scale the throttle accordingly.
```

Comment:
Throttling is disabled in the previous line, skip setting it up altogether?

Reply:
Still good to set the default values I think, incase a user turns throttling on.